### DISCUSSION TITLE
Could you please optimize code for RAM 32GB? there is 70% free space.

### DISCUSSION THREAD

**Discussion by meofather** [madMAx43v3r/chia-plotter#420] [General]
Could you please optimize code for RAM 32GB? there is 70% free space. Can you optimize it as possible?

**Comment by SebMoore** [comment]
If you have free ram space, just decrease the buckets used. This will increase ram usage. Try 64 buckets.

**Comment by madMAx43v3r** [comment]
Just using more RAM does not mean it will be faster...
